Brick-Studios/BeastArena
# In-game advertisement

# Description
I added a new "object" in the JSON for in-game advertisement. This is based on a frame with specific dimensions.

## Screenshot(s)
![image](https://user-images.githubusercontent.com/27761285/70450852-b7de9d80-1aa4-11ea-9b06-85a670404b69.png)

# How can this be tested?
You could try to position billboards yourself but this is a pain in the ass.

What I do need you to verify:
- [ ] You can see the advertisement in level2.json
- [ ] You can see the advertisement in box.json. This means that the default image works correctly.

# Checklist:
